index.js

```diff
@@ -0,0 +1,80 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let className = document.querySelector("#className");
+  let teacherName = document.querySelector("#teacherName");
+  let addClass = document.querySelector("#addClass");
+  let enrollClass = document.querySelector("#studentClass");
+  let enrollName = document.querySelector("#studentName");
+  let enrollAge = document.querySelector("#studentAge");
+  let enrollCity = document.querySelector("#studentCity");
+  let enrollGrade = document.querySelector("#studentGrade");
+  let classList = document.querySelector("#classList");
+  let cityList = document.querySelector("#cityList");
+  let checkFailing = document.querySelector("#checkFailing");
+  let listSubmit = document.querySelector("#listSubmit");
+  let form1 = document.querySelector("#one");
+  let form2 = document.querySelector("#two");
+  let form3 = document.querySelector("#three");
+  let div1 = document.querySelector("#divOne");
+  let div2 = document.querySelector("#divTwo");
+  let div3 = document.querySelector("#three");
+
+  form1.addEventListener("submit", async e => {
+    e.preventDefault();
+    try {
+      axios.post(`http://localhost:3000/class/${className.value}/${teacherName.value}`).then(res => {
+          let result = document.createElement("p");
```

Comment:
You don't have to pass request info via query strings, you can just send it in the body of your request:
`let res = axios.post("http://localhost:3000/class", {name: className.value, teacher: teacherName.value})`
You would grab the information from the object you are passing in your app.js file as req.body.name and req.body.teacher.